townsfolk/investigative/juror

```diff
@@ -0,0 +1,11 @@
+**Juror** | Townsfolk Group
+__Basics__
+The Jurors have a secret channel where they can communicate among themselves securely. On odd days, the jury votes to investigate a player.
+If the jury successfuly investigates a non-town role, it is announced.
+__Details__
+The Juror is part of the jury. After Day 0, on odd days, the jury can vote to investigate a player. 
+The jury recieves the result of the investigation at the end of the day. If the result of the investigation shows a non-town role, the role is immediately announced publicly.
+The jury is disbanded if there are 1 or less Jurors left; this means the jury is active only if atleast two Jurors are alive. 
+Any member of the jury, even if they lose their role, can still vote on who to investigate. 
+One of the Jurors that has voted for the jury's target is chosen as the investigator, who performs the investigations. 
```

Comment:
should it be added that it is randomly chosen from the ones who voted or can the jury choose? how is this impacted when a juror has lost their role but gets chosen to investegate, since imo they cant investiage anymore since they arent that role anymore.

Reply:
Oh yeah, I forgot to add random.

Reply:
think for consistency Jurors that lost their role would still be able to vote and investigate.

Reply:
yea, probably the for the best